### PR TITLE
First Version of Lore

### PR DESCRIPTION
**Pull request by antoniomf97**
Here we present the first version of the Stickian Lore. This is just an initial draft.

### CODE REVIEW

File: documents/LORE.md
Comment (antoniomf97) [inline documents/LORE.md]: Will the Madking become undead or simply die? I still don't have many ideas for this.

File: documents/LORE.md
Comment (antoniomf97) [inline documents/LORE.md]: Maybe this changed too quickly? Ofc it will be more detailed in the complete version.

File: documents/LORE.md
Comment (luisferreira32) [reply documents/LORE.md]: I  think he could just die 🤷 
We can also throw in some mambo jambo of the rubber soldiers vengeful soul is what made the undead rize, and that they can be somehow "put at rest"

File: documents/LORE.md
Comment (luisferreira32) [reply documents/LORE.md]: Well, yea, you should have at least a sentence before this saying there were some stick colonies fighting bravely to survive (...) Will you let them thrive 😉  ?

File: documents/LORE.md
Comment (luisferreira32) [inline documents/LORE.md]: Should not be topics, this is a story, at most it's a summary. So drop down the numbers (for the official md you are going to merge here) and keep a nice summary of our story here.

**Review by luisferreira32 — COMMENTED** [review]
Still a lot of sentences that seem off. Some grammar mistakes, and the necessity of a flow (it should be a summary not topics). The idea however, seems great!

You could also add some spice, for example:
* give a better reason for king Madstick to raise the army (or phrase the current one better, it seems that it needs some improvement)
* have the rubber king appear as the undead who overthrows the madstick
* call this Second Great War (of the Rubberhand 😉 )
* create some hocus pocus reason for the undead to rise (and a hocus pocus logic on how they can be killed - otherwise, how would players fight against them to begin with?)

File: documents/LORE.md
Comment (antoniomf97) [reply documents/LORE.md]: Yeah something like that would be nice

**Comment by luisferreira32** [comment]
Proposal for your summary, worked from your topic points:

Many, many years ago, the Rubberian people were defeated at the Great War of the Rubberhand. Across the now Rubberfree land all stick-realms achieved life in peace, focused on prosperity and growth. However, not all kingdoms prospered together. King Ladstick, the III,  was falling to despair. As the main general of the united realms, hero in the war, and decorated as the strongest general, he was left to rule a barren kingdom ravaged by the war. As he saw the coward neighbour kingdoms prosper from his kingdom's sacrifice, his jealousy increased. Instead of ruling his own kingdom he retreated into his inner chambers and plotter revenge. The land got poorer and the disparities bigger, people were left without ruler, as King Ladstick did not exercise the law. 
One day, the king, now called by many King Madstick, ventured into the Forest of The Thousand Sticks, in a quest to find the dark wizard whom he thought could be the answer to his plan. The king knew the wizard would heed his call, as the wizard was in debt to the Ladstick family, but the risk of his request stroke fear in his heart nonetheless. He steeled his resolve as this would be done for the benefit of the kingdom, or so he thought...
The wizard, in vain, attempted to warn the king about the dangers of the request, but the king could see no other way out at this point. An army of undead, built from the rubber remains of the Great War, was the final demand. And so, with the spirit of the Mythical Eraser, the dark wizard travelled to the Rubberian graveyard, where no stickman would ever go, and managed to breath a second life to the Rubberian people raising a fearsome army that could conquer the world. These undead warriors, pink in skin, meant terror to every stickman, and they were an army ready for battle.
In a resentful rage, the Madstick drove his newly found army to his prospering neighbouring countries, and the first stickities were conquered. However, when you dance with death, you're just inviting it sooner to your house... One night, the king saw in his dreams Rubber Isho, the XVI, the last emperor of the Rubberian people, casually saying while slowly erasing Madstick's existence:
  "How nice, a second chance to erase these stickmen of existence"
and he never woke up again...
And so the kingdom fell, by jealousy to greed, and a misled ruler. With the terrorizing Rubberian undead, advancing in crushing waves, destroying the life of peaceful stickmen, what is there to do? As a last resistance for the stickmen, the last colonies held strong in the cardinal points of the world, fighting bravely to survive! Under YOUR rule, will they be able to thrive?
Will you help them reach victory, or will you fall into misery? The choice is yours.


Edit: for grammar

**Comment by luisferreira32** [comment]
<details>
 <summary>Possible spoiler if this thread is accepted </summary>
We could also have some twists like:

* The mage was controlling the undead rubber king in the end
* The start of the rubber war was the stickmen repressing the rubber people (peaceful rubbers) for being different

+weird stuff+
</details>

File: documents/LORE.md
Comment (antoniomf97) [reply documents/LORE.md]: Before making the first draft it is important to define the story key points, and one way of doing that it's with topics. Moreover this is a WIP for the draft. But yes, this would eventually become a summary on the complete version. Sorry if I was not much explicit.

Comment (bgonilho) [reply documents/LORE.md]: @luisferreira32 if you were paying attention to the meetings you'd know.

File: documents/LORE.md
Comment (bgonilho) [reply documents/LORE.md]: Undead could be for a future event #keeptheidea

File: documents/LORE.md
Comment (bgonilho) [inline documents/LORE.md]: how many years tho

Comment (antoniomf97) [reply documents/LORE.md]: I guess that for now that we can just define time relatively to events. However, later on this might be important to define specific dates (to avoid inconsistencies).

**Comment by antoniomf97** [comment]
Although I like this summary as a version 0, I believe that the concept of the Madking vanishing could have some extra work, or should work differently... Maybe the wizard was half-rubber, half-stickman and wanted to get his revenge?

**Comment by antoniomf97** [comment]
> In a resentful rage, the Madstick drove his newly found army to his prospering neighbouring countries, and the first stickities were conquered. However, when you dance with death, you're just inviting it sooner to your house... One night, the king saw in his dreams Rubber Isho, the XVI, the last emperor of the Rubberian people, casually saying while slowly erasing Madstick's existence:
> "How nice, a second chance to erase these stickmen of existence"
> and he never woke up again...

I would redo this by following one of the two ideas below:
1. "In a resentful rage and thirst for power, the Madstick drove his newly found army to his prospering neighbouring countries, and the first stick-cities were conquered. He was becoming unstoppable, however, his greed was blind to the wizard's warnings... The more he controlled his soldiers, the more his power was getting drained, hence he was becoming weaker and weaker. One day, he became so weak that he fainted, prepared to give up his soul to his army. At this point, the undead army joined forces to bring the king, their god, back from the dead, but he wouldn't be the same... He was full of dark power. He was no longer the Madstick King... He rose as the undead Madking, ruthless and stronger than he ever was."
2. "In a resentful rage and thirst for power, the Madstick drove his newly found army to his prospering neighbouring countries, and the first stick-cities were conquered. He was becoming unstoppable, however, he was blind to the wizard's warnings... His reckless greed led him to raise from the dead the once-loved Rubber Isho, the XVI, the last emperor of the Rubberian people. This undead rubber emperor wanted his revenge, and the Madstick king was on his way. So, the Rubber Isho mercilessly slew the Madstick king in front of all the undead army, and claimed his crown as the undead emperor."

If you have other ideas for possible endings, feel free to share them!

**Comment by luisferreira32** [comment]
Both seem pretty cool! I'd be leaning to 2.

File: documents/LORE.md
Comment (luisferreira32) [inline documents/LORE.md]: I would not put a "complete version" just yet, as that could be expanded in chapters and books in the se_lore repo

Comment (antoniomf97) [reply documents/LORE.md]: Yes, i will replace it by "Summary" or "Short Version"

**Review by luisferreira32 — APPROVED** [review]
Remove the `## Topics` section before merging, they were for discussion... right?

**Pull request merged**
